plugin/hpsa/hpsa.py

```diff
@@ -231,6 +231,17 @@ def _disk_status_of(hp_disk, flag_free):
 
     return disk_status
 
+def _disk_link_type_of(hp_disk):
+    disk_interface = hp_disk['Interface Type']
+    if disk_interface == 'SATA':
+        return Disk.LINK_TYPE_ATA
+    elif disk_interface == 'Solid State SATA':
+        return Disk.LINK_TYPE_ATA
+    elif disk_interface == 'SAS':
+        return Disk.LINK_TYPE_SAS
+
```

Comment:
Hey @cathay4t, for reasons explained elsewhere I'm still unable to actually test this on a Solid State SAS drive, but take a look at this link for an example of a Solid State SAS output from hpssacli: http://serverfault.com/questions/586626/hp-ssd-on-proliant-dl360p-gen8-p420i-controller-no-trim